emulator/src/components/hdw-tft/hdw-tft.c

```diff
@@ -229,7 +229,12 @@ void drawDisplayTft(fnBackgroundDrawCallback_t fnBackgroundDrawCallback)
                     int dstY  = ((y * displayMult) + mY);
                     int pxIdx = (dstY * (TFT_WIDTH * displayMult)) + dstX;
 
-                    uint32_t color = paletteColorsEmu[frameBuffer[(y * TFT_WIDTH) + x]];
+                    int colorolor = frameBuffer[(y * TFT_WIDTH) + x];
+                    if(colorolor == 216){
+                        colorolor = c500;
```

Comment:
Is this drawing a red pixel as a warning that the transparent value leaked into the framebuffer?